src/Modules/OrchardCore.Commerce.Payment/Abstractions/IPaymentProvider.cs

```diff
@@ -34,7 +34,7 @@ public interface IPaymentProvider
     /// <summary>
     /// Validates the data POSTed to the <see cref="PaymentController.Validate"/> action.
     /// </summary>
-    Task ValidateAsync(IUpdateModelAccessor updateModelAccessor, string? shoppingCartId) => Task.CompletedTask;
+    Task ValidateAsync(IUpdateModelAccessor updateModelAccessor, string? shoppingCartId, string? paymentIntentId = null) => Task.CompletedTask;
```

Comment:
Payment Intent is a Stripe-specific term, it should not be used in OrchardCore.Commerce.Payment.Abstractions which applies to all payment providers. In this case it's enough to rename it to `paymentId`.